doc.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 The Gidari Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+package gidari
```

Comment:
```suggestion
//	http://www.apache.org/licenses/LICENSE-2.0

// Package gidari provides a "web-to-storage" API for batch querying web APIs and persisting the resulting data.
package gidari
```

Reply:
Added in https://github.com/alpstable/gidari/pull/384/commits/30e7ecba31c8e1486509efb1641edde9747a57bc